pandas/tests/frame/test_arithmetic.py

```diff
@@ -500,7 +499,9 @@ def test_floordiv_axis0(self):
         tm.assert_frame_equal(result2, expected)
 
     @pytest.mark.parametrize("opname", ["floordiv", "pow"])
-    def test_floordiv_axis0_numexpr_path(self, opname, request):
+    def test_floordiv_axis0_numexpr_path(
```

Comment:
there's a decent chance that with the numexpr fixture (i forget what its called) this test is entirely unnecessary

Reply:
Ah good point. With `switch_numexpr_min_elements` it looks like `test_floordiv_axis0` will make this redundant